runtimes/huggingface/mlserver_huggingface/common.py

```diff
@@ -94,3 +87,33 @@ def parse_parameters_from_env() -> Dict:
                     reason="MICROSERVICE_BAD_PARAMETER",
                 )
     return parsed_parameters
+
+
+def load_pipeline_from_settings(hf_settings: HuggingFaceSettings) -> Pipeline:
+    """
+    TODO
+    """
+    # TODO: Support URI for locally downloaded artifacts
+    # uri = model_parameters.uri
+    model = hf_settings.pretrained_model
+    tokenizer = hf_settings.pretrained_tokenizer
+
+    if model and not tokenizer:
+        tokenizer = model
+
+    if hf_settings.optimum_model:
+        print(f"optimum cache {TRANSFORMER_CACHE_DIR}")
```

Comment:
nit: should we use `logger.debug` for this one?

Reply:
Good catch - totally forgot to remove this one, just updated